Dockerfile

```diff
@@ -51,44 +53,48 @@ LABEL org.label-schema.schema-version="1.0" \
       org.label-schema.vcs-url="https://github.com/inventree/InvenTree.git" \
       org.label-schema.vcs-ref=${commit_tag}
 
-# RUN apt-get upgrade && apt-get update
-RUN apt-get update
-
-# Install required system packages
-RUN apt-get install -y  --no-install-recommends \
-    git gcc g++ gettext gnupg libffi-dev libssl-dev \
-    # Weasyprint requirements : https://doc.courtbouillon.org/weasyprint/stable/first_steps.html#debian-11
-    poppler-utils libpango-1.0-0 libpangoft2-1.0-0 \
+RUN apk add --no-cache \
+    git gettext py-cryptography \
```

Comment:
Not sure we need git any more

Reply:
What about to install a plugin from a remote git repo?

Reply:
I do not think it is a good idea to push that way to install plugins; discovery is slower and the support for nested plugins is limited; pip as a distribution way is way cleaner

Reply:
I would argue to keep git and add a big note somewhere to the plugin dev
section. Sometimes you just want to test something and then git is easier.

Reply:
> pip as a distribution way is way cleaner 

Absolutely agreed, but there may be users who have their own plugins not available via pip?